backend/api/Database/Context/InitDb.cs

```diff
@@ -51,9 +52,16 @@ private static List<Installation> GetInstallations()
                 InstallationCode = "JSV"
             };
 
+            var installation2 = new Installation
+            {
+                Id = Guid.NewGuid().ToString(),
+                Name = "Kårstø",
+                InstallationCode = "KAA"
```

Comment:
Should we utilize Huldra or general names?

Reply:
This should be addressed in another PR

Reply:
This has been addressed in another PR so we should now use Huldra here